packages/driver/src/cy/retries.ts

```diff
@@ -73,6 +73,18 @@ export const create = (Cypress, state, timeout, clearTimeout, whenStable, finish
       }).message
 
       const retryErrProps = modifyErrMsg(error, prependMsg, (msg1, msg2) => {
+        const autOrigin = Cypress.state('autOrigin')
+        const commandOrigin = window.location.origin
+
+        if (autOrigin && commandOrigin !== autOrigin) {
```

Comment:
Could also consider using `cors.urlOriginsMatch()`

Reply:
This is a good idea